README.md

```diff
@@ -8,6 +8,38 @@ To do this run `make run-validator` to run the validator and then run `make vali
 
 **Schema file names must use snake case to be compatible with runner**
 
+## Generating translated schemas
```

Comment:
Since author will be using this repo, we need to add a section about how to set up and install the dependencies. So how to install python (pyenv), how to install poetry, and how to install the dependencies.